### ISSUE TITLE
[UX2.0] other_ucse_profile_parcel some optional fields are marked as required

### ISSUE THREAD

**Issue by tzarski0** [CiscoDevNet/terraform-provider-sdwan#320]
The following paramters are marked as required but should be optional as they are not required when access_port_dedicated is configured:
access_port_shared_failover_type
access_port_shared_type

The following paramters are marked as required but should be optional as they can be "_variable" too:
default_gateway
ipv4_address

**Comment by tzarski0** [comment]
When I try to configure UCSE feature with access_port_dedicated = true, it still fails saying the access_port_shared_type is required:
```
│   with module.sdwan.sdwan_other_ucse_feature.other_ucse_feature["basic_other-ucse"],
│   on ../terraform-sdwan-nac-sdwan/sdwan_features.tf line 24, in resource "sdwan_other_ucse_feature" "other_ucse_feature":
│   24:   access_port_shared_type          = try(each.value.ucse.cimc_access_port_shared_type, null)
│ 
│ Must set a configuration value for the access_port_shared_type attribute as the provider has marked it
│ as required.
│ 
│ Refer to the provider documentation or contact the provider developers for additional information about
│ configurable attributes that are required.
```

**Comment by seconroy** [comment]
Resolved in release 0.5.1
